### ISSUE TITLE
from Done to Noted (OnNotes like)

### ISSUE THREAD

**Issue by fastrizwaan** [done-devs/done#107]
Dear Eduardo Flores,

Done is a beautiful program, it's souce can also be used to create a Note Taking application like Microsoft OneNote

https://www.microsoft.com/en-in/microsoft-365/onenote/digital-note-taking-app

Please consider it. Thanks.

**Comment by edfloreshz** [comment]
Hello! Thanks for your interest in Done 😄 

Do you think [Paper](https://flathub.org/apps/io.posidon.Paper) can satisfy this requirement? Is a very beautiful app.

**Comment by fastrizwaan** [comment]
Agreed that it is a beautiful app, but functionality is not working properly. Like Search for keywords does not work. Also, it only supports `Book->Note` whereas, I wish for `Book->Chapter/Topic->Note`

Also it is not updated for almost a year now. That's why I requested for creating a notes app, seeing that you have excellent flair for usability and beauty.

Done has the 3 level support,

`Computer->Lists->List` which is similar to what I'd like to see in a notebook `Book->Chapter->Note`.  Please consider a notes app based on Done's shared code base and paper's richtext or markdown code. 


Thank you.

**Comment by edfloreshz** [comment]
I will consider it, although I cannot make any promises as my time is already rather limited these days. 

But it sounds like a good idea 💡

**Comment by edfloreshz** [comment]
I think this is the closest thing to Done regarding notes.

https://flathub.org/apps/org.gnome.World.Iotas
